src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -222,6 +214,14 @@ public SolrQueryResponse search(
         }
         
         List<DataverseMetadataBlockFacet> metadataBlockFacets = new LinkedList<>();
+        
+        /*
+         * We talked about this in slack on 2021-09-14, Users can see objects on draft/unpublished 
+         *  if the owner gives permissions to all users so it makes sense to expose this facet 
+         *  to all users. The request of this change started because the order of the facets were 
+         *  changed with the PR #9635 and this was unintended.
+         */
+         solrQuery.addFacetField(SearchFields.PUBLICATION_STATUS);
```

Comment:
Should still be inside the if(addFacets)?

Reply:
Done! Thanks a lot, good catch 🐟